src/main/java/core/basesyntax/FileRead.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.util.List;
+
+public interface FileRead {
+    List<String> readFile(File file);
```

Comment:
```suggestion
    List<String> readFromFile(String fromFileName);
```
you should create a File object inside of this method